WakaTime.py

```diff
@@ -103,6 +104,48 @@ def invoke(self, context, event):
         return context.window_manager.invoke_props_dialog(self)
 
 
+# Addon prefs
+class WakaTimePreferences(bpy.types.AddonPreferences):
+    _default_always_overwrite_projectname = 0
+    _default_chars = '1234567890._'
+    _default_prefix = ''
+    _default_postfix = ''
+    _default_use_project_folder = 0
+    bl_idname = __name__
+    _def_chars = _default_chars if _default_chars else "<empty>"
+    _def_prefix = _default_prefix if _default_prefix else "<empty>"
+    _def_postfix = _default_postfix if _default_postfix else "<empty>"
```

Comment:
I didn't find anything special about `_def_...` class attributes in `AddonPreferences` docs, so I assume the sole purpose of these are for the `description`-s below.
If so, I propose to remove them and change the descriptions as follows (in the next comments) 
```suggestion
```